File: Frontend/revolve/src/app/model/checkout/checkoutRequest.ts
export class CheckoutRequest{
    public token:String
    public fullname: String;
    public provinceID: number;
    public phone: String;
    public password: String;

    constructor(token:String, name:String, province: number, phone:String, password:String){
        this.token = token;
        this.fullname = name;
        this.provinceID = province;
        this.phone = phone;
        this.password = password;
    }
}

File: Backend/Lulus.ViewModels/Cart/CheckoutRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lulus.ViewModels.Cart
{
    public class CheckoutRequest
    {
        public string Token { get; set; }
        public string Fullname { get; set; }
        public int ProvinceID { get; set; }
        public string Phone { get; set; }
        public string Password { get; set; }
    }
}
